PerformanceTests/Sources/PerformanceTestScript/Util/Benchmark.swift

```diff
@@ -0,0 +1,69 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+struct BenchmarkResult: Codable {
+    let name: String
+    let description: String
+    let unit: String
```

Comment:
Should we maybe add an enum for the supported unit types that map to raw string values so it's easier to maintain going forward? Atm supported cases are `Operations/Second, Bits/Second, Megabits/Second, Gigabits/Second, Seconds, Milliseconds, Microseconds, Nanoseconds, Bytes, Megabytes, Gigabytes, Percentage`